src/main/java/com/epam/test_generator/controllers/caze/response/CaseDTO.java

```diff
@@ -136,6 +136,10 @@ public void setStatus(Status status) {
         this.status = status;
     }
 
+    public String getDisplayedStatus() {
+        return status.getStatusName();
```

Comment:
Can we avoid using `Status` in this dto and just set `statusName` and `statusDisplayName` values here?

Reply:
We can do it. Do you suggest in all response DTO use String instead Status and delete additional statusDisplayName method?